libs/judicial-system/types/src/lib/digitalMailboxCase.ts

```diff
@@ -0,0 +1,11 @@
+export interface IndictmentCaseData {
```

Comment:
Why did you put this interface here instead of having it as a type in the digital mailbox api?